packages/postgres-database/tests/products/test_utils_products.py

```diff
@@ -5,7 +5,7 @@
 
 
 import asyncio
-from typing import Callable
+from collections.abc import Callable
 
```

Comment:
Q: just curious, why this change? thanks

Reply:
ruff